test/core/gas/MarketOrderOtherOfferList.t.sol

```diff
@@ -234,6 +234,8 @@ contract ExternalMarketOrderOtherOfferList_WithMultipleOffersAtManyTicks is Tick
     mgv.marketOrderByLogPrice(_olKey, _logPrice, 2 ** 96, false);
     gas_();
     (, MgvStructs.LocalPacked local) = mgv.config(_olKey);
-    assertLt(_logPrice, LogPriceLib.fromTick(local.bestTick(), _olKey.tickScale));
+    if (!local.level3().isEmpty()) {
```

Comment:
How come this is necessary now? Does it mean some assumption in the tests no longer holds?

Reply:
see #544

Reply:
#554 I believe